source/common/http/conn_manager_impl.h

```diff
@@ -191,6 +184,16 @@ class ConnectionManagerImpl : Logger::Loggable<Logger::Id::http>,
     // Http::RequestDecoder
     void decodeHeaders(RequestHeaderMapPtr&& headers, bool end_stream) override;
     void decodeTrailers(RequestTrailerMapPtr&& trailers) override;
+    const StreamInfo::StreamInfo& streamInfo() const override {
+      return filter_manager_.streamInfo();
```

Comment:
When is this allocated? Can this ever be null between returning the decoder to the codec and this being called?

Reply:
stream info is a member variable of filter_manager_ and initialized on construction:
https://github.com/envoyproxy/envoy/blob/main/source/common/http/filter_manager.h#L989

Ditto filter_manager_ and ActiveStream
https://github.com/envoyproxy/envoy/blob/main/source/common/http/conn_manager_impl.h#L351

Reply:
OK thank for confirming.